docs/source/author/topics.rst

```diff
@@ -0,0 +1,685 @@
+Topics application
+##############################################################################
+
+The topics application is the main focus of the CS Unplugged website, as it
+contains the majority of educational material for the project.
+
+.. contents:: Contents
+  :local:
+
+Topics overview
+==============================================================================
+
+A general overview of the topics application can be described in the following
```

Comment:
This is defined in the 'Project Structure' doc:

> Django contains ‘applications’ which are Python packages that provide some set of features. Each large part/chunk of the CS Unplugged is a separate application. The project currently contains the following applications:

Is it worth adding a pointer?